src/common-elements/panels.ts

```diff
@@ -1,5 +1,6 @@
 import { SECTION_ATTR } from '../services/MenuStore';
 import styled, { media, withProps } from '../styled-components';
+import { darken } from 'polished';
```

Comment:
From Travis-CI:

ERROR: /home/travis/build/Rebilly/ReDoc/src/common-elements/panels.ts[3, 1]: Import sources within a group must be alphabetized.